javascript/src/URDFLoader.js

```diff
@@ -271,6 +276,45 @@ class URDFLoader {
             obj.colliders = colliderMap;
             obj.visual = visualMap;
 
+            // Link up mimic joints
+            jointList.forEach(j => {
+
+                if (j.mimicJoint) {
+
+                    if (jointMap.hasOwnProperty(j.mimicJoint)) {
+
+                        jointMap[j.mimicJoint].mimicJoints.push(j);
+
+                    }
+
+                }
+
+            });
+
+            // Detect infinite loops of mimic joints
+            jointList.forEach(j => {
+
+                const uniqueJoints = new Set();
+                const iterFunction = joint => {
+
+                    if (uniqueJoints.has(joint)) {
+
+                        throw new Error('Detected an infinite loop of mimic joints');
```

Comment:
Nit: Can we add `URDFLoader:` to the beginning of the error and add a period to the end of the error.

Reply:
Sure, done